File: Vocal.Model/Request/SendMessageRequest.cs
using System;
using System.Collections.Generic;

namespace Vocal.Model.Request
{
    public class SendMessageRequest : Request
    {
        public string IdTalk { get; set; }
        public string Content { get; set; }
        public DateTime SentTime { get; set; }
        public int MessageType { get; set; } 
        public string IdSender { get; set; }
        public List<string> IdsRecipient { get; set; }
        public string IdMessageParent { get; set; }
    }
}

File: Vocal.MobileApp/src/models/request/sendMessageRequest.ts
import {Request} from './request';

export class SendMessageRequest extends Request {

    content : string;
    sentTime: Date;
    duration: number;
    platform: string;
    IdTalk: string;
    messageType : number;
    idSender : string;
    idsRecipient : Array<string> = [];
    Lang: string;

    constructor(idSender, IdTalk, content, messageType, idsRecipient) {
        super();
        this.idSender = idSender;
        this.content = content;
        this.IdTalk = IdTalk;
        this.messageType = messageType;
        this.idsRecipient = idsRecipient;
        this.sentTime =new Date();
    }
}